Description: enable building of gettext translations.
Author: Andreas Metzler <ametzler@debian.org>
Last-Update: 2023-10-26
Origin: vendor

--- a/configure.ac
+++ b/configure.ac
@@ -64,11 +64,10 @@ AC_PROG_CPP
 AM_PROG_CC_C_O
 PKG_PROG_PKG_CONFIG
 AM_PATH_PYTHON([3.6],, [:])
 AM_MISSING_PROG([CASTXML], [castxml])
 
-LINGUAS=""
 AM_GNU_GETTEXT([external], [need-ngettext])
 AM_GNU_GETTEXT_VERSION([0.19.8])
 m4_ifdef([AM_GNU_GETTEXT_REQUIRE_VERSION],[
 AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.8])
 ])
